.travis.yml

```diff
@@ -90,14 +86,15 @@ before_script:
   - composer install --prefer-dist --no-interaction
   - if [[ $PHPCS = 3 ]]; then composer remove --dev cakephp/cakephp-codesniffer; fi
   - if [[ $PHPCS = 3 ]]; then composer require --dev squizlabs/php_codesniffer='3.0.*@dev' cakephp/cakephp-codesniffer='3.0.*@dev'; fi
+  - printf "yes\n" | pecl install apcu
```

Comment:
I haven't noticed that .travis.yml already contains installation of APCu elsewhere so I changed the commit: https://github.com/cakephp/cakephp/pull/10044/commits/b8c60da7957aaf835a24f9fb83bcdab5bae16890